photon-core/src/main/java/org/photonvision/common/configuration/NeuralNetworkModelManager.java

```diff
@@ -358,6 +368,10 @@ public void discoverModels() {
         try {
             Files.walk(modelsDirectory.toPath())
                     .filter(Files::isRegularFile)
+                    .filter(
+                            path ->
+                                    path.toString().endsWith(Family.RKNN.extension())
```

Comment:
Nit: consider iterating over every family, rather than hard coding the two families here? Otherwise happy with this approach